packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -188,6 +188,7 @@ export interface RumContext {
     session: {
       plan: RumSessionPlan
     }
+    browser_sdk_version: string
```

Comment:
should not we update rum-event-format too?

Reply:
Indeed.